.github/workflows/build-vault-oss.yml

```
@@ -44,10 +44,13 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: ${{ inputs.go-version }}
+      - name: Read node version from .nvmrc
+        id: node-version
+        run: echo "version=$(cat ./ui/.nvmrc)" >> "$GITHUB_OUTPUT"
       - name: Set up node and yarn
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: ${{ steps.node-version.outputs.version}}
```

Hey, I think we can slightly streamline this workflow by using the [Node version file option](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file). 

The suggestion below means that we wouldn't need the `Read node version from .nvmrc` step. We use something similar when setting up Go and had no issues so far, you can see an example [here](https://github.com/hashicorp/vault/blob/main/.github/workflows/test-go.yml#L85).

```suggestion
          node-version-file: './ui/.nvmrc'
```

---

@kubawi that is definitely a more streamlined option and since that option supports using `package.json` that's even better. Thanks for the suggestion! The `test-enos-scenario-ui.yml` workflow was doing something similar for both the go and node versions so I updated it there as well if you wouldn't mind having a look.

---

Yep, looks good. I left an approval on the PR 👍